packages/core/src/utilities/actorCheck.ts

```diff
@@ -0,0 +1,24 @@
+import { Types } from '..';
+
+type actorTypes = 'vtkActor' | 'vtkVolume' | 'vtkImageSlice';
+
+/**
+ * Checks if a vtk Actor is an image actor (vtkVolume or vtkImageSlice) otherwise returns false.
+ *
+ * @param actor - actor
+ * @returns A boolean value.
+ */
+export function isImageActor(actorEntry: Types.ActorEntry): boolean {
+  return (
```

Comment:
Consider removing the outer parentheses.

Reply:
It is the prettier, so we can discuss it later.

Reply:
ok np...